src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -186,12 +185,32 @@ public static Version GetInstalledAsmVersion2(IEnumerable<Version> versions, ref
         /// 
         public static void PreloadAsmFromPath(string preloaderLocation, string asmLocation)
         {
-            if (string.IsNullOrEmpty(preloaderLocation) || !Directory.Exists(preloaderLocation))
+            // this will with be empty, the originally requested preloaderLocation or a remapped location
```

Comment:
you mean to say "this will be either empty or ..."

Reply:
yes, thanks